elem/elem.gen.go

```diff
@@ -970,7 +970,7 @@ func WordBreakOpportunity(markup ...vecty.Markup) *vecty.Element {
 	return e
 }
 
-// Heading elements implement six levels of document headings, <h1> is the most important and <h6> is the least. A heading element briefly describes the topic of the section it introduces. Heading information may be used by user agents, for example, to construct a table of contents for a document automatically.
+// Heading elements implement six levels of document headings, <h1> is the most important and <h6> is the least. A heading element briefly describes the topic of the section it introduces. Heading information may be used by user agents, for example, to construct a table of contents for a document automatically( just like the fixed sider bar of this page on the right).
```

Comment:
The upstream page isn't updating.

- Fixed on the source page: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements 
- Won't update on the primary page, ctrl+f for `( just like`, https://developer.mozilla.org/en-US/docs/Web/HTML/Element
- Doing shift+refresh will queue the page for an update.. but even after finished it just doesn't update.